examples/image-processing/src/pages/index.js

```diff
@@ -306,14 +306,19 @@ export const pageQuery = graphql`
     sizes: imageSharp(id: { regex: "/fecolormatrix-kanye-west.jpg/" }) {
       sizes(
         duotone: { highlight: "#f00e2e", shadow: "#192550" }
+        trace: {
```

Comment:
@KyleAMathews Unsure about arg naming – should this rather be `traceSVG` or even matching the field name (`tracedSVG` … which I dislike for no apparent reason ;-))?

Reply:
it should be a noun — so `tracedSVG`. GraphQL is declarative. You say — "I want X!" And X is given to you.

Reply:
👍  (even if in this case it's "I want to configure X" ;-))

Reply:
Oh my bad — this is the arguments not the field. Yeah, `traceSVG` then is right.